integration-tests/src/test/java/com/pip/chatbot/integration/tests/tests/weather/weatherAdmin.feature

```diff
@@ -0,0 +1,192 @@
+Feature: Restaurants Api
+
+  Background:
+    * url baseUrl
+    * def initDatabase =
+    """
+    function() {
+      var DbUtils = Java.type('com.pip.chatbot.integration.tests.WeatherDbUtils');
+      var db = new DbUtils(dbConfig);
+      db.initWeatherData();
+     return db;
+    }
+    """
+    * def db = callonce initDatabase
+
+    * json cities = read('classpath:weather/cities.json')
+    * json cityModify = read('classpath:weather/citiesModification.json')
+    * json countries = read('classpath:weather/countries.json')
+    * json countryModify = read('classpath:weather/countriesModification.json')
+
+    * configure afterFeature =
+    """
+    function(){
+      db.clearWeatherData();
+    }
+    """
+
+    * def correctAuthorization = callonce read('basic-auth.js') {user: #(user), password: #(password)}
+    * def wrongAuthorization = call read('basic-auth.js') {user: #(user), password: #(incorrectPassword)}
+    * header Authorization = correctAuthorization
+
+  Scenario: get countries
+    Given path '/admin/forecasts/countries'
+    When method GET
+    Then status 200
+    And match $ == countries
+
+  Scenario: insert and verify country
+    Given path '/admin/forecasts/countries'
+    And request countryModify.countryUpdate
```

Comment:
Zróbmy country do inserta, bo to nie jest czytelne, że do insert używamy update, a do update delete